Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1

```diff
@@ -7535,7 +7535,7 @@ Function Set-PinnedApplication {
 
 				Invoke-Verb -FilePath $FilePath -Verb $PinVerbAction
 			}
-			ElseIf ($Action -in 'PintoTaskbar', 'UnpinfromTaskbar') {
```

Comment:
Please add a comment on why we're not using -in but instead .Contains() so the devs later on know it.